### DISCUSSION TITLE
All Comments in a single separate file?

### DISCUSSION THREAD

**Discussion by Sandhu1472** [linanwx/aosr#19] [General]
Hi, @linanwx
I don't know if that is possible but can you make this plugin to save all the comments in a separate master file so as to reduce the clutter in actual notes.
Regards.

**Comment by linanwx** [comment]
Hi @Sandhu1472,

Thank you for your suggestion regarding saving all the comments in a separate master file to reduce clutter in the actual notes. I appreciate your input and see the value in keeping the notes tidy.

I want you to know that I'm currently considering this idea and exploring various options to implement it without causing any issues with the binding of cards and their data information. It's a bit complex, and I want to make sure to find the best solution that aligns with the functionality and user experience of the plugin.

I'll keep you updated as I make progress on this. Feel free to share any additional thoughts or suggestions you might have.

Regards,
linanwx

**Comment by linanwx** [comment]
I'm pleased to inform you that we have already implemented this feature in version 1.1.1. Now, all data is saved in a separate db file instead of in the notes, which should greatly reduce the clutter in the notes. Thanks again for your feedback and suggestion.
